dist/stickyfill.js

```diff
@@ -371,8 +371,8 @@
         updateScrollPos();
         initAll();
 
```

Comment:
@krzksz thanks for the input! I'll test it out on [18F/stickyfill](https://github.com/18F/stickyfill) and see if it helps with older browsers!